### ISSUE TITLE
Tiled gallery

### ISSUE THREAD

**Issue by Android63** [NHSLeadership/nightingale-wp#71]
Format a tiled gallery of images so they appear as a grid

**Comment by Android63** [comment]
Nightingale doesn't have a tiled gallery. Should it? Let's discuss @cehwitham and @Pro-Paul

**Comment by cehwitham** [comment]
This is the default WordPress gallery layout: https://codex.wordpress.org/The_WordPress_Gallery

We've also used tiled galleries on a previous setup: https://jetpack.com/support/tiled-galleries/

Not sure how much use case there is for galleries over single images but I think articles like graduation ceremonies would benefit.

What are your thoughts @Pro-Paul ?

**Comment by Pro-Paul** [comment]
Hi,

Just to weigh in here quickly.

There may well be a use case here but not one I can see as being a high enough priority at the moment to be implemented in version 1. 

I would suggest that this can be reviewed properly for implementation once we have a closer close to implementation of v1 and look to add the feature in or even replace one we do some user testing with the first iteration of v1.

So I suggested to keep on the back burner for now.

**Comment by Android63** [comment]
I'm happy to put this on the back-burner but what if a site admin adds a gallery? At the moment, it won't look good. Is that OK @cehwitham?

**Comment by cehwitham** [comment]
How bad does it look @Android63 ?

**Comment by Android63** [comment]
The images are in one column @cehwitham, no matter how many columns are set:

![screenshot-nightingalewpdev local-2018-02-02-15-14-15-365](https://user-images.githubusercontent.com/1991226/35740146-c7b6bc02-082b-11e8-8df1-4b1b4dc2a5b9.png)

**Comment by cehwitham** [comment]
I agree with Paul, we don't want to make this a big thing but I wonder if there's a quick win mapping the existing column classes to the WP markup? Wouldn't want to spend more than an hour on it.

**Comment by Android63** [comment]
That's my think too @cehwitham. I'll take a look...
